packages/google_sign_in/google_sign_in_ios/darwin/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -270,30 +270,19 @@ - (void)addScopes:(NSArray<NSString *> *)scopes
 #endif
 }
 
-/// @return @c nil if GoogleService-Info.plist not found and clientId is not provided.
-- (GIDConfiguration *)configurationWithClientIdArgument:(id)clientIDArg
-                                 serverClientIdArgument:(id)serverClientIDArg
-                                   hostedDomainArgument:(id)hostedDomainArg {
-  NSString *clientID;
-  BOOL hasDynamicClientId = [clientIDArg isKindOfClass:[NSString class]];
-  if (hasDynamicClientId) {
-    clientID = clientIDArg;
-  } else if (self.googleServiceProperties) {
-    clientID = self.googleServiceProperties[kClientIdKey];
-  } else {
-    // We couldn't resolve a clientId, without which we cannot create a GIDConfiguration.
+/// @return @c nil if GoogleService-Info.plist not found and runtimeClientIdentifier is not
+/// provided.
+- (GIDConfiguration *)configurationWithClientIdentifier:(NSString *)runtimeClientIdentifier
+                                 serverClientIdentifier:(NSString *)runtimeServerClientIdentifier
+                                           hostedDomain:(id)hostedDomain {
```

Comment:
Why is hostedDomain still using id?

Reply:
No idea, I thought I changed all three. Fixed!